### ISSUE TITLE
Deprecate CUDA macro

### ISSUE THREAD

**Issue by evaneschneider** [cholla-hydro/cholla#241]

**Comment by bcaddy** [comment]
Does this mean we should just remove `CUDA` macros as we come across them? Or do it all in one fell swoop?

**Comment by evaneschneider** [comment]
Yes, with the exception of course of any code that controls the CUDA / HIP compile conversion.

**Comment by bcaddy** [comment]
Resolved in #360
